app/controllers/api/v1x0/mixins/index_mixin.rb

```diff
@@ -16,8 +16,8 @@ def collection(base_query)
           render :json => ManageIQ::API::Common::PaginatedResponse.new(
             :base_query => filtered(scoped(base_query)),
             :request    => request,
-            :limit      => pagination_params[:limit],
-            :offset     => pagination_params[:offset]
+            :limit      => params[:limit],
```

Comment:
@syncrou @eclarizio 
This is now going to come from the manageiq-api-common gem. Waiting for my PR to get merged https://github.com/ManageIQ/manageiq-api-common/pull/99
As part of the parameter mixing there is a function called pagination_limit and pagination_offset.